src/GovUk.Education.ExploreEducationStatistics.Common/Comparers/ListValueComparer.cs

```diff
@@ -0,0 +1,14 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Comparers;
+
+public class ListValueComparer<TValue>(IEqualityComparer<TValue>? equalityComparer = null)
+    : ValueComparer<List<TValue>>(
+        (c1, c2) => c1!.SequenceEqual(c2!, equalityComparer),
```

Comment:
Are we always assured we're going to be given non-null Lists when using this Comparer, or should we build in some null-checking here?  I expect that where we've used this pattern previously in the codebase, we've always known that we're guaranteed to have non-null Lists, but now this is generic and reusable maybe we should build null safety in.